x/plans/keeper/migrations.go

```diff
@@ -22,3 +25,39 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
+	planIndices := m.keeper.GetAllPlanIndices(ctx)
+
+	for _, planIndex := range planIndices {
+		var oldPlanStruct v3.Plan
```

Comment:
instead of "old" and "new", use the version, e.g. `var plan_v3 v3.Plan`
(also below)

Reply:
done